src/shared/UI/spinner/Spinner.component.tsx

```diff
@@ -17,7 +17,7 @@ export const Spinner: FC<SpinnerProps> = ({ variant = 'small' }) => {
 
   return (
```

Comment:
You have 2 returns in one component. I guess you should try "conditional rendering" code style to keep code cleaner

Reply:
There can be more than two returns in the code. And if it will be I'll refactor this code to `switch...case`. The second `return case` is the return by default. And may I know - what the problem can be with two returns in one component?

Reply:
> There can be more than two returns in the code. And if it will be I'll refactor this code to `switch...case`. The second `return case` is the return by default. And may I know - what the problem can be with two returns in one component?

I didn't think about potential `switch...case` using, thanks for explaining.
I would say that using a single return is quite a common idiom in React code, but multiple return also isn't wrong.